source/_patterns/01-atoms/button/__tests__/button.test.js

```diff
@@ -0,0 +1,37 @@
+const $ = require('jquery');
```

Comment:
Tests are a part of our design system, and we don't expect these to move when / if demo folders and _twig-components move?

Reply:
Tests stay close to components. Bundles get deployed. Won't be affected by demo reshuffling.